src/test/java/de/tum/in/www1/artemis/AssessmentComplaintIntegrationTest.java

```diff
@@ -303,6 +303,25 @@ public void getComplaintsForAssessmentDashboard_sameTutorAsAssessor_studentInfoH
         });
     }
 
+    @Test
+    @WithMockUser(username = "instructor1", roles = "INSTRUCTOR")
+    public void getComplaintsForAssessmentDashboard_testRun() throws Exception {
+        complaint.setParticipant(database.getUserByLogin("instructor1"));
+        complaintRepo.save(complaint);
+
+        final var params = new LinkedMultiValueMap<String, String>();
+        params.add("complaintType", ComplaintType.COMPLAINT.name());
+        final var complaints = request.getList("/api/exercises/" + modelingExercise.getId() + "/complaints-for-test-run-dashboard", HttpStatus.OK, Complaint.class, params);
+
+        complaints.forEach(compl -> {
+            final var participation = (StudentParticipation) compl.getResult().getParticipation();
+            assertThat(participation.getStudent()).as("No student information").isNull();
+            assertThat(compl.getParticipant()).as("No student information").isNull();
+            assertThat(participation.getExercise()).as("No additional exercise information").isNull();
+            assertThat(compl.getResultBeforeComplaint()).as("No old result information").isNull();
+        });
```

Comment:
This does not look like a useful test for complaints of a test run. I would assume that you intend to check here that there is a complaint that fulfills the constraints listed here. But this test would pass if `complaints` is empty, it seems more reasonable to me to check that we get the single, correct complaint from the REST-API back (id, content).

Reply:
I'll check it